newrelic/exporter_test.go

```diff
@@ -152,8 +154,11 @@ func TestEndToEndTracer(t *testing.T) {
 		t.Fatalf("failed to instantiate exporter: %v", err)
 	}
 
+	r := resource.NewWithAttributes(semconv.ServiceNameKey.String(serviceName))
+
 	tracerProvider := sdktrace.NewTracerProvider(
 		sdktrace.WithBatcher(e, sdktrace.WithBatchTimeout(15), sdktrace.WithMaxExportBatchSize(10)),
+		sdktrace.WithResource(r),
```

Comment:
I think it'd be nice to modify the example app like this as well (adding ServiceName resource here: https://github.com/newrelic/opentelemetry-exporter-go/blob/master/examples/simple/main.go#L57).

Reply:
@prmsrswt sure, thanks

Reply:
I think this version should go up, after the new release
https://github.com/newrelic/opentelemetry-exporter-go/blob/844f0922993392b95d57217a1f962e039fe2c1fa/examples/simple/go.mod#L7

Reply:
a new `go.opentelemetry.io/otel` v0.19.0 has been released, should this PR be updated to the latest one?

Reply:
Indeed. Merging this PR soon as a stepping stone to 0.19.0.